client/my-sites/plans/current-plan/thank-you-header/index.js

```diff
@@ -0,0 +1,90 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { get, isFinite, noop } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import getCurrentPlanPurchaseId from 'state/selectors/get-current-plan-purchase-id';
+import ProgressBar from 'components/progress-bar';
+import QuerySitePurchases from 'components/data/query-site-purchases';
+import { getByPurchaseId } from 'state/purchases/selectors';
+import { getPlan } from 'lib/plans';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSitePlanSlug } from 'state/sites/selectors';
+
+export class PlanSetupHeader extends Component {
+	static propTypes = {
+		progressComplete: PropTypes.number,
+		progressTotal: PropTypes.number,
+	};
+
+	render() {
+		const { moment, purchaseExpiryDate, planName, siteId, translate } = this.props;
+		const duration = purchaseExpiryDate
+			? moment.duration( moment().diff( purchaseExpiryDate ) ).humanize()
+			: null;
+		return (
+			<Card className="thank-you-header">
+				<QuerySitePurchases siteId={ siteId } />
+				<img
+					className="thank-you-header__illustration"
+					alt=""
+					aria-hidden="true"
+					src="/calypso/images/illustrations/fireworks.svg"
+				/>
+				<h1 className="thank-you-header__title">{ translate( 'Thank you for your purchase!' ) }</h1>
```

Comment:
Does that really deserve a `<h1 />`? I would expect a `<h2 />` or a `<h3 />` instead.

Reply:
I took a look at this page and similar pages and `h1` seemed to fit pretty well with the page hierarchy:

![h* hierarchy](https://user-images.githubusercontent.com/841763/45144849-8dc26f80-b1bf-11e8-8e83-f32dd5a691ed.png)

Reply:
Hmm, well that felt odd. I've always known that it's recommended that a page has only 1 `<h1>` tag, and that corresponds to the actual page title, and not to any of the page section titles. In that context, it was unexpected to see multiple `<h1>` tags. 

I don't have any strong feelings about it though, so I'll leave the final decision to you.

Reply:
I took a quick look and didn't find any specific rules around limiting `<h1>` to one per page.

Given that there are already offenders on this page and we're not going to fix the entire hierarchy in this PR –it's problematic that the `h3` feature cards "descend" from the `h1` "Professional Plan Features"– I'll leave this.